service/history/timer_queue_standby_task_executor.go

```diff
@@ -426,6 +425,38 @@ func (t *timerQueueStandbyTaskExecutor) executeWorkflowTimeoutTask(
 	)
 }
 
+func (t *timerQueueStandbyTaskExecutor) executeWorkflowExecutionTimeoutTask(
+	ctx context.Context,
+	timerTask *tasks.WorkflowExecutionTimeoutTask,
+) error {
+	actionFn := func(
```

Comment:
could you put some comment here what this is doing

Reply:
done.